lib/records.ncl

```diff
@@ -338,7 +377,7 @@ let error_lib = import "./error.ncl" in
           let bad_field_names =
             std.record.fields x
             |> std.array.filter (fun k => !(is_match k))
-            |> std.array.map (fun k => "`${k}`")
+            |> std.array.map (fun k => m%""${k}""%)
```

Comment:
The interpolation doesn't seem to be right :sweat_smile: 
```suggestion
            |> std.array.map (fun k => m%""%{k}""%)
```

Reply:
Clearly I've been writing too much python :sweat_smile: